domains/luked.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "luked177",
+        "email": ""
+    },
+    "record": {
+        "CNAME": "cv-sand-zeta.vercel.app/"
```

CNAME can't contain `/`
```suggestion
        "CNAME": "cv-sand-zeta.vercel.app"
```